include/mbedtls/dhm.h

```diff
@@ -298,6 +298,7 @@ int mbedtls_dhm_calc_secret( mbedtls_dhm_context *ctx,
 void mbedtls_dhm_free( mbedtls_dhm_context *ctx );
 
 #if defined(MBEDTLS_ASN1_PARSE_C)
+/** \ingroup x509_module */
```

Comment:
This is actually a regression since this reference to x509 was removed from crypto but added back by the branch reconciliation. It's a downside of the unremove strategy: the unremoval includes a few minor things that we actually wanted to remove independently of the repository separation.

I think anything non-cosmetic at this stage would either show up visibly as a test coverage regression (e.g. something bad happening to `all.sh`) or as a CI failure. I'll fix this one, but if there are other similar issues, they were acceptable for some time before the repository split, so they should still be acceptable now.

Reply:
Fixed in https://github.com/ARMmbed/mbed-crypto/pull/384